.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,53 @@
+<!--- Provide a general summary of your changes in the Title above -->
+<!--Read comments, before committing pull request read checklist again
+
+# Checklist:
+
+- [ ] I have performed a self-review of my own code,
+- [ ] I have commented my code, particularly in hard-to-understand areas,
+- [ ] I have made corresponding changes to the documentation,
+- [ ] I have added tests that prove my fix is effective or that my feature works,
+- [ ] New and existing unit tests pass locally with my changes
+- [ ] Title include "WIP" if work is in progress.
+-->
+
+Resolves # <!--fill issue number-->
+
+## Description
+
+<!--- Describe your changes in detail -->
+
+### Motivation and Context
+
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+<!--- If there's a Figma file or similar spec, please link to to the issue. List any dependencies that are required for this change. (gems, js libraries, etc.)
+
+Include anything else we should know about.-->
+
+### Type of change
+
+<!-- Please delete options that are not relevant. -->
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+- [ ] This change requires a documentation update
+- [ ] Documentation update
+
+### How Has This Been Tested?
+
+<!--- Please describe in detail how you tested your changes so that a reviewer can reproduce the results. -->
+<!--- Include details of your testing environment, and the tests you've run your self -->
+<!--- see how your change affects other areas of the code, etc. -->
+
+### How Will This Be Deployed?
```

Comment:
i think in 99% of cases there will be no changes to deployment - maybe we could add this as a checkbox in type of change instead, i.e. "requires change to deployment proces"?

Reply:
I like this idea. Will do!

Reply:
Resolved in [453f671](https://github.com/geeksforsocialchange/PlaceCal/pull/2630/commits/453f671991dcc3d3ffd32332616652d07ad4d695)